Exerscise_3/main.cpp

```diff
@@ -86,19 +87,54 @@ int main(int args, char **argv){
             std::cerr << ia.what() << std::endl;
             return EXIT_FAILURE;
         }
-        if (write_data_to_file("../Exam.txt", exam_vector)) {
+        if (args == 2) {
+            if (write_data_to_file("../Exam.txt", exam_vector)) {
+                return EXIT_FAILURE;
+            }
+            if (write_data_to_file("../Cource.txt", cource_vector)) {
+                return EXIT_FAILURE;
+            }
+            if (write_data_to_file("../Student.txt", student_vector)) {
+                return EXIT_FAILURE;
+            }
+            if (write_data_to_file("../Teacher.txt", teacher_vector)) {
+                return EXIT_FAILURE;
+            }
+            return EXIT_SUCCESS;
+        } else if (args > 2 and argv[2]==std::string("SHOW")) {
+            std::vector<std::pair<std::string, std::vector<std::pair<int, std::unique_ptr<IRecord>>>>> container;
+            if (args==3 or argv[3] == std::string("STUDENTS")) {
```

Comment:
I know that `and` and `or` work in C++, but please avoid them (: That's a common approach that you use `&&` and `||`